CppPluginSupport/src/MfxAttribute.cpp

```diff
@@ -98,3 +126,13 @@ void MfxAttribute::CopyFrom(MfxAttribute& other, int start, int count)
     FetchProperties(destinationProps);
     copyAttributeData(destinationProps, sourceProps, start, count);
 }
+
+void MfxAttribute::ForwardFrom(MfxAttribute &other) {
+    MfxAttributeProps sourceProps, destinationProps;
+    other.FetchProperties(sourceProps);
+
+    destinationProps = sourceProps;
+    destinationProps.isOwner = false;
```

Comment:
Shouldn't we also copy the buffer access info (stride, componentCount, type)?

Reply:
That's what the `destinationProps = sourceProps;` does, it copies all fields fo the struct :) I'm just setting `isOwner` to false to avoid double-free in case that the attribute we're forwarding from was allocated by the host via `meshAlloc()` (as it can happen in Blender with loose edge meshes after my last pull request).

Reply:
Come to think of it, there is an edge case I didn't think of - calling the forwarding only works if the buffer pointer is there, ie. you cannot create a new output attribute (`isOwner = true, data = NULL`), then forward it to another attribute, and then call `meshAlloc()` - only the first attribute will get the buffer, the second will still be NULL.

I'm not sure there's any legitimate use case for that, but it could me mentioned in the docstring.

Reply:
Oh sure my bad I read too quickly! For your edge case I don't think of any good reason to do so, that would mean forwarding an output to another output, right?

Reply:
Yes, it would be forwarding between output attributes, which seems silly.